sesman/chansrv/chansrv_fuse.c

```diff
@@ -2850,9 +2873,9 @@ static void xfuse_cb_setattr(fuse_req_t req, fuse_ino_t ino, struct stat *attr,
 
     if (to_set & FUSE_SET_ATTR_SIZE)
     {
-        log_debug("previous file size: %d", attr->st_size);
+        log_debug("previous file size: %lld", (long long)attr->st_size);
```

Comment:
Why cannot you use the `z` modifier here? Please provide some context for your environment.